### ISSUE TITLE
Vehicle can't be entered even when the whole crew is dead

### ISSUE THREAD

**Issue by IrLED** [A3Antistasi/antistasiofficial#58]
I don't know if that is native ARMA defect or somehow related to issue #57 (Empty groups are not discarded from HighCommand list). I think these bugs correlate (maybe i'm wrong)
To check the crew - ACE menu - passengers helps.
Possible example:
USMC QRF is called (to get access to MH-6, actually exploit), then heli is damaged by armor or MG. Pilot lands heli, crew got shot. Dead pilot is still in cockpit, player can't enter heli.

**Comment by StefArma** [comment]
It's intended that you can't access NATO/USMC vehicles

if this is happening with enemy vehicles then it's a bug

**Comment by IrLED** [comment]
Hmm. Then the only explanation why people were using MH-6 is that it was spawned by admin. Later that day USMC QRF was called (with the intention to acquire  helis, context of radio comms), but with little success. 
The other day i personally shot down Mi8 with QRF personnel (by heavy MG), heli landed, engines working, crew is dead, no way to get inside.

**Comment by colmaniac** [comment]
Have you tried "unlocking" the vehicle with the console?  I have had the same thing too,  4 man truck, shot all of them in it, non had the chance to get out,  got to the drivers door and locked.  Used console command to  unlock and was able to use it,  Also tried it in a USMC truck still manned in drivers seat. doors locked, I just unlocked it.  I manged to get into the truck, but a few seconds got kicked out,  il try again next time,but shoot driver first lol.
